fern/fern.config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "organization": "didx",
-    "version": "0.9.5"
+  "organization": "didx",
+  "version": "0.9.6-rc1"
```

Comment:
The version here is for the Fern CLI. I have upgraded it with some fixes so that it will work with your OpenAPI.